src/scenekit.cs

```diff
@@ -1967,7 +1967,7 @@ interface SCNSceneLoadingOptions {
 	[MacCatalyst (13, 1)]
 	[BaseType (typeof (NSObject))]
 	interface SCNNode : SCNAnimatable, SCNBoundingVolume, SCNActionable, NSCopying, NSSecureCoding
-#if IOS || TVOS
+#if (IOS || TVOS) && !XAMCORE_6_0 // Conformance Removed in Xcode 16.1
```

Comment:
No need to wait that long:

```suggestion
#if (IOS || TVOS) && !XAMCORE_5_0 // Conformance Removed in Xcode 16.1
```